src/ts/ssh-keys/pkcs8KeyFormatter.ts

```diff
@@ -516,7 +516,8 @@ export class Pkcs8KeyFormatter implements KeyFormatter {
 		iterations: number,
 		keyLength: number,
 	): Promise<Buffer> {
-		const useWebCrypto = !!(typeof crypto === 'object' && crypto.subtle);
+		const useWebCrypto = typeof window !== 'undefined' &&
```

Comment:
Refactor into a helper function? It's also used in other place.

Reply:
Done for 2 of them. The 3rd is in another package so they can't share without making it a public API.